frontend/src/Pages/LessonAnswerPage.tsx

```diff
@@ -0,0 +1,67 @@
+import ChoicesButton from "../components/common/ChoicesButton";
+import Done from "../components/common/Done";
+import Navbar from "../components/common/Navbar";
+import QuizHeader from "../components/common/QuizHeader";
+import DummyTest from "../dummytest";
```

Comment:
For dummy data, please store it in the exact location where it is being used since this is temporary and removable in the future

apply to all related